SV-COMP 2021

lucasccordeiro opened the pull request:
Signed-off-by: Lucas Cordeiro <lucasccordeiro@gmail.com>

lucasccordeiro committed:
enabled incremental context-bounded verification

Signed-off-by: Lucas Cordeiro <lucasccordeiro@gmail.com>

rafaelsamenezes commented:
I think we should mark this as Draft and don't merge it. I will configure the CI to create a release for the competition

rafaelsamenezes committed:
added svcomp build

rafaelsamenezes committed:
Update svcomp.yml

rafaelsamenezes committed:
Added python version

rafaelsamenezes committed:
Fixed property issue

mikhailramalho committed:
Add claim if we are trying to free an object allocated with alloca

Signed-off-by: Mikhail R. Gadelha <mikhail.ramalho@gmail.com>

mikhailramalho committed:
Disable inductive step when checking termination

Signed-off-by: Mikhail R. Gadelha <mikhail.ramalho@gmail.com>

rafaelsamenezes committed:
added --incremental-cb only to incr strategy

rafaelsamenezes committed:
marked everything with incremental except reach

rafaelsamenezes committed:
marked incremental-cb for reach only

Commit:
added concurrency option

lucasccordeiro committed:
adopted slightly modified ISO 8601 format

Signed-off-by: Lucas Cordeiro <lucasccordeiro@gmail.com>

rafaelsamenezes committed:
added no-slice for concurrency

Commit:
added is_const_expr method

This methods checks if both arguments of a byte_extract_id are
constants, if so then it will apply the propagation rules;

rafaelsamenezes committed:
Added create_new_thread into assert (witness)

mikhailramalho committed:
Fix building wrong cex for boolector

Signed-off-by: Mikhail R. Gadelha <mikhail.ramalho@gmail.com>

rafaelsamenezes committed:
added extra information on the version

This should make easier to track which version we submitted

rafaelsamenezes committed:
incremental is no longer overriding kinduction

rafaelsamenezes committed:
marked false-memory as unknown

rafaelsamenezes committed:
Added byte_* operations on witness filter

mikhailramalho commented:
Can we close this PR?

lucasccordeiro commented:
Yes. The most significant changes have already been integrated into the master branch.
